secondIife.js

```diff
@@ -0,0 +1,37 @@
+var CarLot = (function (NewCarLot) {
+		// grab all inputs globally 
+		var carHolder = document.getElementsByClassName("carCard");
+		var newInputField = document.getElementById("inputField");
+		var descriptionInput = document.getElementById("Description");
+		var targetToEdit = "";
+		// itterates through the car cards 
+	NewCarLot.activateEvents = function() {
+	    for (var i = 0; i < carHolder.length; i++) {
+	      var containerEl = carHolder[i];
+	      // add event listener to car cards
+	      containerEl.addEventListener("click", function(event) {
+	            targetToEdit = this.childNodes[4];
```

Comment:
I am guessing I would need to select the description by name instead of node position but not exactly sure how to execute that